ddtrace/tracer.py

```diff
@@ -500,6 +498,11 @@ def configure(
                     msg = "- DATADOG TRACER DIAGNOSTIC - %s" % agent_error
                     self._log_compat(logging.WARNING, msg)
 
+        # Workaround for remote-config with gevent
+        # TODO: Remove when it is fixed
+        if isinstance(self._writer, AgentWriter):
+            self._writer.write([])
```

Comment:
I don't think this is going to do anything?

https://github.com/DataDog/dd-trace-py/blob/123ba1d3a/ddtrace/internal/writer.py#L502-L505

Reply:
Yeah, I copied it from: https://github.com/DataDog/dd-trace-py/blob/03a1e2b2bca1a2b28dc064caf90e15ac302f6d50/ddtrace/internal/debug.py#L59

Testing it with a real django application (with gunicorn and gevent), that call avoids getting a `ImproperlyConfigured` django error, as the wsgi file was not even read otherwise

Reply:
🤦🏻 yeah, /me needs more ☕ to review code I guess